### ISSUE TITLE
Run full pipeline with real image and output results. 

### ISSUE THREAD

**Issue by ngovanmao** [FCCLab/Neurosurgeon-chinese-rebuild#7]
- [ ] Load the model weights from pre-trained model for ImageNet data
- [ ] Revise script to receive image continuously (i.e., from the test set loaded inside the RAM) and feed into the Head model.
- [ ] Send {the activations output, and Inference delay of the Head model} to the Tail model (at the N6 server).
- [ ] Revise script in the Server to receive these inputs and output the results {class, and E2E delay (with broken down into Head, Transmission, and Tail delays)}.
